go.mod

```diff
@@ -154,6 +154,9 @@ require (
 	nhooyr.io/websocket v1.8.6 // indirect
 )
 
+// Short-lived replace for an outstanding PR of log.
```

Comment:
What we do now is to use the commit of the PR to create a pseudo version, so it limits the amount of PRs.

Reply:
We have a script to update it everywhere btw `./scripts/go-update-dep-all.sh cosmossdk.io/log@commit`

Reply:
Thanks, I just pushed 0ed8a417440ef27729baab7dee45908ded1c5429 which references commit 74c894f from this PR.

Reply:
This one is still on main, actually.